android/app/src/main/java/updated/mysterium/vpn/ui/top/up/card/summary/CardSummaryActivity.kt

```diff
@@ -35,7 +35,8 @@ class CardSummaryActivity : BaseActivity() {
         const val GATEWAY_EXTRA_KEY = "GATEWAY_EXTRA_KEY"
         private const val HTML_MIME_TYPE = "text/html"
         private const val ENCODING = "utf-8"
-        private const val paymentCallbackUrl = "payment/stripe/redirect"
+        private const val stripePaymentCallbackUrl = "payment/stripe/redirect"
+        private const val paypalPaymentCallbackUrl = "payment/paypal/redirect"
```

Comment:
Let's move url constants to separate object file because this activity has a lot of data information:
```
object AppConstants {
    
    object Payments {
         const val STRIPE_CALLBACK_URL = "payment/stripe/redirect"
         const val PAYPAL_CALLBACK_URL = "payment/paypal/redirect"
    }
}
```
Put this file in common package. And in future will try to move into this file most of out constants